pr-deploy/src/app.ts

```diff
@@ -70,7 +76,8 @@ function initializeRouter(app: Application) {
     response.send({status: 200});
   };
 
-  router.post('/headshas/:headSha/:result', initialize);
+  router.post('/headshas/:headSha/:result(success)/:externalId', initialize);
```

Comment:
From reading https://github.com/ampproject/amphtml/pull/33774/files#diff-f456adc9b34fa8977425b08583a46b2106b4208392eaa06e7b72b6995137ea16R93-R94, am I correct that `externalId` here corresponds to the CircleCI `jobId` (a.k.a. `CIRCLE_BUILD_NUM`)? If so, would it be clearer to call this section `jobId` instead of `externalId`?

Reply:
`externalId` is how this is called in the GitHub API - I don't mind calling it either way, lmk

Reply:
Oh, if it's a well-known part of an API, okay to leave it as-is. (Maybe add a link to `pr-deploy/redacted.env` for documentation?)